### DISCUSSION TITLE
How to switch from maximizing to minimizing f(x)?

### DISCUSSION THREAD

**Discussion by SorenSc** [josejimenezluna/pyGPGO#32] [Q&A] [answered]
Since I have to minimize the outcome (mean absolute error) of a regression performed by a Random Forest, I simply negated the returned outcome/error/mean absolute error to make it work. But it feels like there might be a better solution. Can anyone point me to that?

Btw.: I was looking for a better place to post this question, but couldn't find a proper community on StackExchange.

**Comment by josejimenezluna** [comment, marked as answer]
maximing f is the same as minimizing -f

**Reply by SorenSc** [reply]
I know, but is there some parameter in the code so that I can switch from maximization to minimization of f instead of negating my outcome?

**Reply by josejimenezluna** [reply]
Not currently.

**Reply by SorenSc** [reply]
Okay, thanks!
